scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -1187,10 +1201,31 @@ private void endJob(JobData jobData, TerminationData terminationData, InternalTa
         // update job and tasks events list and send it to front-end
         updateJobInSchedulerState(job, event);
 
+        cleanJobSignals(jobId);
+
         jlogger.info(job.getId(), "finished (" + jobStatus + ")");
         jlogger.close(job.getId());
     }
 
+    private void cleanJobSignals(JobId jobId) {
+        try {
+            if (synchronizationInternal != null) {
+
+                synchronizationInternal.createChannelIfAbsent(SIGNAL_ORIGINATOR,
+                                                              SIGNAL_TASK_ID,
+                                                              PASchedulerProperties.SCHEDULER_SIGNALS_CHANNEL.getValueAsString(),
+                                                              true);
+
+                synchronizationInternal.remove(SIGNAL_ORIGINATOR,
+                                               SIGNAL_TASK_ID,
+                                               PASchedulerProperties.SCHEDULER_SIGNALS_CHANNEL.getValueAsString(),
+                                               jobId.value());
+            }
+        } catch (IOException | InvalidChannelException e) {
+            jlogger.warn(jobId, "Could not clear the job" + jobId + " from the signals channel");
```

Comment:
why jlogger and add `e` at the end

Reply:
It already has 2 params, and I added a custom message instead of e